scripts/react-native-helper.js

```diff
@@ -54,15 +61,25 @@ module.exports = {
     common.run(`npm install --registry ${registryUrl}`, true)
 
     // Install notifier
-    const command = `npm install @bugsnag/react-native@${version} --registry ${registryUrl}`
+    let command = `npm install @bugsnag/react-native@${version}  --registry ${registryUrl}`
+    common.run(command, true)
+
+    // Install navigation tracker
+    command = `npm install @bugsnag/plugin-react-navigation@${version} --registry ${registryUrl}`
```

Comment:
How's the `plugin-react-native-navigation` going to fit in?

Reply:
That's a good question, I'd assumed they were the same module until now